docs/consume-packages/Finding-and-Choosing-Packages.md

```diff
@@ -147,7 +147,8 @@ For the MSBuild equivalent to setting the license in the `.nuspec`, take a look
 NuGet package search works the same on nuget.org, from the NuGet CLI, and within the NuGet Package Manager extension in
 Visual Studio. In general, search is applied to keywords as well as package descriptions.
```

Comment:
This may be misleading. What do you think of this?

```suggestion
If nuget.org is your package source, you can use search syntax to further refine your search results. This works on nuget,org, in the NuGet CLI, or within the NuGet Package Manager UI in Visual Studio:
```